ContributeDocumentation.md

```diff
@@ -0,0 +1,88 @@
+# Contributing Application Documentation Changes
+
+This document describes how to contribute changes to the ISIS Application Documentation, which is available online for the most recent ISIS release at: https://isis.astrogeology.usgs.gov/Application/index.html
+
+# Getting Started
+Documentation for ISIS applications is located within the ISIS source code and written in XML. The XML is processed to create the documentation that can be viewed on the ISIS website at: https://isis.astrogeology.usgs.gov/Application/index.html
+
+For many documentation contributions, very little understanding of XML is required. Changes in existing wording or additional information can be added where appropriate in the current documentation without needing to change any of the XML. 
```

Comment:
Unless I'm misunderstanding, the second sentence 
isn't true.

Reply:
Thanks for pointing this out! My intention was to communicate that you could edit the wording of the documentation without editing/modifying the xml tags themselves because this originally was aimed at a larger (and potentially less-technical) audience, but you're right and this needs to be fixed. Not sure how to update the wording yet, or maybe I'll just take this out.